cookbooks/bcpc/recipes/zabbix-head.rb

```diff
@@ -202,6 +207,7 @@
   EOH
   not_if "test -r /etc/apache2/sites-enabled/zabbix-web"
   notifies :restart, "service[apache2]", :delayed
+  notifies :run, "ruby_block[run_state_apache2_restart]", :immediate
```

Comment:
This obviates the previous line which should make sure zabbix-server is restarted towards the end of the chef-client run. If we want a resource to trigger an immediate restart of zabbix-server, it would be good to use an immediate restart notification instead of keeping this additional state.